staging/src/kubevirt.io/client-go/kubevirt/typed/core/v1/async.go

```diff
@@ -202,34 +203,30 @@ func RequestFromConfig(config *rest.Config, resource, name, namespace, subresour
 	return req, nil
 }
 
-// enrichError checks the response body for a k8s Status object and extracts the error from it.
-// TODO the k8s http REST client has very sophisticated handling, investigate on how we can reuse it
-func enrichError(httpErr error, resp *http.Response) error {
+// EnrichError checks the response body for a k8s Status object and extracts the error from it.
+func EnrichError(httpErr error, resp *http.Response) error {
 	if resp == nil {
 		return httpErr
 	}
-	httpErr = fmt.Errorf("Can't connect to websocket (%d): %s\n", resp.StatusCode, httpErr)
-	status := &metav1.Status{}
 
-	if resp.Header.Get("Content-Type") != "application/json" {
-		return httpErr
-	}
-	// decode, but if the result is Status return that as an error instead.
+	httpErr = fmt.Errorf("Can't connect to websocket (%d): %s", resp.StatusCode, httpErr)
```

Comment:
```suggestion
	httpErr = fmt.Errorf("Can't connect to websocket (%d): %w", resp.StatusCode, httpErr)
```

Would it make sense to wrap the error properly?